.docs/index.md

```diff
@@ -3,50 +3,24 @@
 Openly serious about storage
 
 ---
-REX-Ray delivers persistent storage access for container runtimes, such as
-Docker and Mesos, and provides an easy interface for enabling advanced storage
-functionality across common storage, virtualization and cloud platforms. For
-example, here's how to list the volumes for a VirtualBox VM running Linux with
-REX-Ray:
+REX-Ray is an open source storage volume manager for container runtimes, such as Docker and Mesos. REX-Ray enables stateful applications, such as databases, key/value stores or any containerized app, to persist and maintain it's data after the life cycle of the container has ended. Built-in High Availability allows orchestrators such as [Docker Swarm](https://docs.docker.com/engine/swarm/), [Kubernetes](http://kubernetes.io/), and [Mesos Frameworks](http://mesos.apache.org/) like [Marathon](https://mesosphere.github.io/marathon/) to automatically orchestrate storage tasks between hosts in a cluster.
```

Comment:
> REX-Ray is an open source storage volume manager for container runtimes, such as Docker and Mesos.

Hmmm, how about:

> REX-Ray is an open source, storage management solution, designed to support container runtimes such as Docker and Mesos.

I get the following point, but I kind of think it can be tightened up, you know? 

> REX-Ray enables stateful applications, such as databases, key/value stores or any containerized app, to persist and maintain it's data after the life cycle of the container has ended.

What do you think of the following?

> REX-Ray turns any containerized application into a stateful one, enabling data persistence that lives beyond the life cycle of the container.